docs/ai-coder/tasks.md

```diff
@@ -141,9 +153,15 @@ Coder can automatically generate a name your tasks if you set the `ANTHROPIC_API
 
 If you tried Tasks and decided you don't want to use it, you can hide the Tasks tab by starting `coder server` with the `CODER_HIDE_AI_TASKS=true` environment variable or the `--hide-ai-tasks` flag.
 
-## Command Line Interface
+## Command Line Interface and API Access
+
+Tasks can be managed programmatically through the Coder CLI or REST API:
+
+- **CLI Commands**: See [Tasks CLI](./cli.md) for creating, monitoring, and managing tasks from scripts or CI/CD pipelines
+- **REST API**: See [Tasks API Reference](../reference/api/tasks.md) for HTTP endpoints to integrate task creation into your automation workflows
+- **GitHub Actions**: Use the [create-task-action](https://github.com/coder/create-task-action) to trigger tasks from GitHub workflows
```

Comment:
I would remove this as it feels like a random inclusion to the list. If we want to keep the callout, we can add a new section around "current and planned integrations"